components/engine_traits/src/raft_engine.rs

```diff
@@ -138,6 +138,12 @@ pub trait RaftEngine: RaftEngineReadOnly + Clone + Sync + Send + 'static {
     fn dump_stats(&self) -> Result<String>;
 
     fn get_engine_size(&self) -> Result<u64>;
+
+    /// Visit all available raft groups.
+    fn for_each_raft_group<E, F>(&self, f: &mut F) -> std::result::Result<(), E>
```

Comment:
why this is added on the engine_trait level?

Reply:
I need a way to get all the available raft groups from raft engine, what's your suggestion?